src/components/UserEdit/index.tsx

```diff
@@ -80,6 +80,7 @@ const UserEdit: React.FC = () => {
         permissions: currentPermission,
         email: user?.email,
         avatar: user?.avatar,
+        userType: user?.userType,
```

Comment:
I don't think we should pass user type. It shouldn't be editable by the user anyways. We should instead change the user route on the server end to merge the actual editable values being submitted.

Reply:
Don't the same apply for email and avatar though?

Reply:
Avatar yes, email will be editable for local users. So no.

At some point we could also make avatars editable but not very important.

Reply:
Hm ok. I see your point then.

The issue is not that it isn't merging the result I think, but that the User schema lists userType as a required parameter.

Reply:
Oh, yeah. Ok let's get this merged for now and we can refactor it later.